compiler/llvm/clangUtil.cpp

```diff
@@ -2184,10 +2184,34 @@ void runClang(const char* just_parse_filename) {
   clangOtherArgs.push_back("sys_basic.h");
 
   if (!just_parse_filename) {
+
+    if (localeUsesGPU()) {
+      //create a header file to include header files from the command line
+      std::string genHeaderFilename;
+      genHeaderFilename = genIntermediateFilename("generated_header.h");
```

Comment:
Let's name the file something more descriptive. Perhaps "command-line-includes.h".